llvm/lib/Target/DirectX/DXILShaderFlags.cpp

```diff
@@ -63,20 +106,64 @@ void ComputedShaderFlags::print(raw_ostream &OS) const {
   OS << ";\n";
 }
 
+/// Get the combined shader flag mask of all module functions.
+const ComputedShaderFlags DXILModuleShaderFlagsInfo::getCombinedFlags() const {
```

Comment:
nit: a one-line function like this we should probably declare inline in the header so that it can be inlined at calls.